test/functional/Java16andUp/src/org/openj9/test/foreignMemoryAccess/TestCloseScope0.java

```diff
@@ -111,6 +112,8 @@ public static void closeScopeDuringAccess() throws Throwable {
 
 		Thread t1 = new Thread(()->{
 			try {
+				t1Started = true;
```

Comment:
To be safe `t1Started = true;` should be moved in the `synchronized (synch1) {` block